default.xml

```diff
@@ -14,37 +14,51 @@
 
     <!-- We assume this manifest is in the same path as all the libraries. If you are
      copying this manifest as a template then you probably want the fetch path
-     for this to be '../seL4' instead of '.' -->
-    <remote name="seL4"
-        fetch="."
-    />
+     for this to be '../seL4' instead of '.' added to .repo/local_manifest/ -->
+    <remote name="current" fetch="."/>
```

Comment:
No, I apparently didn't check this diff as closely as I should have and this flew under my radar.
I remembered that the default was ".", but forgot that "." was aliased to seL4 rather than used directly.

overlooked due to mostly rewriting the whole file